install_config/router/default_haproxy_router.adoc

```diff
@@ -207,6 +207,28 @@ $ oadm router region-west -o yaml --images=myrepo/somerouter:mytag \
 ====
 endif::[]
 
+[[concurrent-connections]]
+Setting the Maximum Number of Concurrent Connections::
+
+The default maximum number of connections that can be handled by the router is 2000. 
+You can change that limit depending on your needs.  Too few connections will prevent the
+health check from working which causes unneeded restarts. The system needs to be configured
+to support the number of connections.  The limits shown in 'sysctl fs.nr_open' and 'sysctl fs.file-max'
```

Comment:
`The system needs to be configured to support the number of connections needed by the health check`